leaf-ui/js/onFunctionsNextState.js

```diff
@@ -7,4 +7,26 @@
 // var sliderNextState = document.getElementById("colorResetNextState");
 // sliderNextState.oninput = function() { //changes slider mode and refreshes 
 //     EVONextState.setSliderOption(this.value);
-// }
\ No newline at end of file
+// }
+
+//Only event listeners
+$('#btn-zoom-in').on('click', function(){ zoomIn(analysis.paperScroller); });
+$('#btn-zoom-out').on('click', function(){ zoomOut(analysis.paperScroller); });
+$('#btn-fnt').on('click', function(){ defaultFont(analysis.paper);});
+$('#btn-fnt-up').on('click', function(){  fontUp(analysis.paper);});
+$('#btn-fnt-down').on('click', function(){ fontDown(analysis.paper);});
+$('#inspector-btn-small').on('click', function(){ goToState(); });
+$('#conflictFl').on('click', function(){ add_filter();});
```

Comment:
For all the add_filter() params, since they all call the same function regardless of id, you can probably just use the class name to get it all in one line. Aka $('.filter_checkbox').on('click', function(){add_filter});

Reply:
$('.filter_checkbox').on('click', function(){add_filter}); wasn't working for some reason? You could still select filters but they weren't actually filtering. Not sure why. Should I look more into this?

Reply:
I forgot to put the () after add_filter - could that be it? Either way I would probably look into it more if you don't mind cause it could save a lot of lines of code and simplify stuff down